components/fieldgroup/stories/fieldgroup.stories.js

```diff
@@ -1,35 +1,80 @@
 // Import the component markup template
 import { Template } from "./template";
+import { default as Radio } from "@spectrum-css/radio/stories/radio.stories.js";
 
 export default {
   title: "Field group",
   description: "The Field group component is...",
   component: "Fieldgroup",
   argTypes: {
-    size: {
-      name: "Size",
+    layout: {
+      name: "Layout",
       type: { name: "string", required: true },
       table: {
         type: { summary: "string" },
         category: "Component",
       },
-      options: ["s", "m", "l", "xl"],
+      options: ["vertical", "horizontal"],
       control: "select"
     },
+    labelPosition: {
+      name: "Label Position",
+      type: { name: "string" },
+      table: {
+        type: { summary: "string" },
+        category: "Component",
+      },
+      options: ["top", "side"],
+      control: "select"
+    },
+    isInvalid: {
+      name: "Invalid",
+      type: { name: "boolean" },
+      table: {
+        type: { summary: "boolean" },
+        category: "State",
+      },
+      control: "boolean",
+    },
   },
   args: {
-    rootClass: "spectrum-",
-    size: "m",
+    rootClass: "spectrum-FieldGroup",
+    layout: "vertical",
+    labelPosition: "top",
+    isInvalid: false
   },
   parameters: {
     actions: {
-      handles: []
+      handles: [
+        ...Radio.parameters.actions.handles,
+      ]
     },
     status: {
       type: process.env.MIGRATED_PACKAGES.includes('fieldgroup') ? 'migrated' : undefined
     }
   }
 };
 
-export const Default = Template.bind({});
-Default.args = {};
+export const Vertical = Template.bind({});
+Vertical.args = {
+  layout: "vertical",
+  items: [{
+    id: "1",
+    label: "Radio 1",
+  },{
+    id: "2",
+    label: "Radio 2",
+  }]
+};
+
+export const Horizontal = Template.bind({});
+Horizontal.args = {
```

Comment:
I am noticing we have displayed horizontal and vertical as stories and within the controls table, do we want to remove them from the controls?

Reply:
I took a page out of Cassondra's book for the AssetCard. She has a few stories there but they are all also controls. I don't think it's bad to allow users to have controls, the stories just give them a quick way to visualize a main variation without having to remember to check it by changing it themselves